be/src/runtime/sender_queue.cpp

```diff
@@ -535,6 +537,11 @@ void DataStreamRecvr::PipelineSenderQueue::decrement_senders(int be_number) {
     VLOG_FILE << "decremented senders: fragment_instance_id=" << print_id(_recvr->fragment_instance_id())
               << " node_id=" << _recvr->dest_node_id() << " #senders=" << _num_remaining_senders
               << " be_number=" << be_number;
+    if (UNLIKELY(_num_remaining_senders < 0)) {
```

Comment:
VLOG is enough in 537

Reply:
if the if is reached, there are bugs, should print a log to show this problem.

Reply:
It's not a bug.